source/jormungandr/tests/authentication_tests.py

```diff
@@ -419,8 +422,47 @@ def test_sort_coverage(self):
             assert regions[1]["name"] == 'empty routing'
             assert regions[2]["name"] == 'routing api data'
 
+    def test_coverage_by_coords(self):
+        with user_set(app, FakeUserAuth, 'bobitto'):
+            response = self.query('v1/coverage/{coords}'.format(coords=coords))
+
+            r = get_not_null(response, 'regions')
+
+            region_ids = {region['id']: region for region in r}
```

Comment:
minor you can rewrite those 3 lines: `assert {region['id'] for region in r} == {'main_routing_test'}`